resources/Database/Schema.sql

```diff
@@ -155,11 +156,12 @@ CREATE TABLE `phplist_linktrack` (
 DROP TABLE IF EXISTS `phplist_linktrack_forward`;
 CREATE TABLE `phplist_linktrack_forward` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `url` varchar(255) DEFAULT NULL,
+  `url` varchar(2083) DEFAULT NULL,
+  `urlhash` char(32) DEFAULT NULL,
   `personalise` tinyint(4) DEFAULT '0',
   PRIMARY KEY (`id`),
-  UNIQUE KEY `urlunique` (`url`),
-  KEY `urlindex` (`url`)
+  UNIQUE KEY `urlunique` (`urlhash`),
+  KEY `urlindex` (`url(255)`)
```

Comment:
This looks like a copy'n'paste error. (The `(255)` should not be there.)

Reply:
@oliverklee it is not copy paste error, it's supposed to be that way because the index length has to be specified 255 when varchar length > 255, otherwise it will give an error.

Reply:
Please check: https://github.com/phpList/phplist3/blob/master/public_html/lists/admin/structure.php

Reply:
AFAICS, this currently causes the build to fail. What can we do?

Reply:
@oliverklee I think this is caused by the usage of backticks in the SQL syntax, as I have been doing some syntax checks here: http://sqlfiddle.com and it works fine without using backticks.
Different resources I have checked suggest using them for reserved words only. Thoughts?

Reply:
Whatever works in MySQL on Travis. (The SQL file is only for running the tests.)